### PR TITLE
improve image layer size output when size info not available

### PR DESCRIPTION
**Pull request by juanvallejo**
cc @soltysh @smarterclayton 

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1666583

### CODE REVIEW

File: pkg/oc/cli/image/info/info.go [hunk 1]
```diff
@@ -196,12 +196,22 @@ func describeImage(out io.Writer, image *Image) error {
 	case 1:
 		fmt.Fprintf(w, "Image Size:\t%s\n", units.HumanSize(float64(image.Layers[0].Size)))
 	default:
-		fmt.Fprintf(w, "Image Size:\t%s\n", fmt.Sprintf("%s in %d layers", units.HumanSize(float64(image.Config.Size)), len(image.Layers)))
+		imageSize := fmt.Sprintf("%s in %d layers", units.HumanSize(float64(image.Config.Size)), len(image.Layers))
+		if image.Config.Size == 0 {
+			imageSize = fmt.Sprintf("%d layers (size information not available)", len(image.Layers))
```
Comment (smarterclayton) [inline pkg/oc/cli/image/info/info.go]: This is too verbose.  Just print no size

Comment (smarterclayton) [reply pkg/oc/cli/image/info/info.go]: List the layers

Comment (juanvallejo) [reply pkg/oc/cli/image/info/info.go]: Thanks, updated. Layers are still being printed below:

e.g.
```
Image Size:    7 layers (no size)
Layers:        -- sha256:a02a4930cb5d36f3290eb84f4bfa30668ef2e9fe3a1fb73ec015fc58b9958b17
               -- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
               -- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
               -- sha256:5eeafb18f3a347319cc2f108f61ecf654e4c02761242a5f3f60bbd9fa4eabc44
               -- sha256:2f09d5438e83863c7d01e976ca9983507ed584644e3e22cff4866736669b7fef
               -- sha256:47ba804948df9908e95bb49dca1c771055c65cb197f9c68c34ae8f653395b512
               -- sha256:e9157828eb63c9105fad94bc56adeba441156eeb5d5ab35a352472c3c53cc6d0
```

**Comment by juanvallejo** [comment]
/test e2e-aws

cc @smarterclayton

File: pkg/oc/cli/image/info/info.go [hunk 2]
```diff
@@ -196,12 +196,22 @@ func describeImage(out io.Writer, image *Image) error {
 	case 1:
 		fmt.Fprintf(w, "Image Size:\t%s\n", units.HumanSize(float64(image.Layers[0].Size)))
 	default:
-		fmt.Fprintf(w, "Image Size:\t%s\n", fmt.Sprintf("%s in %d layers", units.HumanSize(float64(image.Config.Size)), len(image.Layers)))
+		imageSize := fmt.Sprintf("%s in %d layers", units.HumanSize(float64(image.Config.Size)), len(image.Layers))
+		if image.Config.Size == 0 {
+			imageSize = fmt.Sprintf("%d layers (no size)", len(image.Layers))
```
Comment (smarterclayton) [inline pkg/oc/cli/image/info/info.go]: (size unavailable)

Comment (juanvallejo) [reply pkg/oc/cli/image/info/info.go]: Thanks, updated:

```
Image Size:    7 layers (size unavailable)
Layers:        -- sha256:a02a4930cb5d36f3290eb84f4bfa30668ef2e9fe3a1fb73ec015fc58b9958b17
               -- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
               -- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
               -- sha256:6d1570ccea24733f8f62a569cde51ebb6d1d80caa8f92c2bc8db5207186c818f
               -- sha256:bed7fc1ba1cdcc4256668e30acc80826804465b9733b941a84cdfb6fe9a1f3f2
               -- sha256:d112385f67ff233dac2b84fd7c6b806c8ff046307e20e1e56478dc8a0de55a32
               -- sha256:615cb848ad62fed9c4943f42f081f2719c28014d9d4ace9c533eef31b6d142ea
```

**Comment by soltysh** [comment]
Closing in favor of https://github.com/openshift/origin/pull/22201

**Pull request closed without merging**